components/eamxx/tests/uncoupled/mam4/mam4_microphysics_standalone.cpp

```diff
@@ -0,0 +1,70 @@
+#include <catch2/catch.hpp>
```

Comment:
Unless you have a reason not to, you could consider removing this file, and using `share/util/eamxx_ad_test.cpp` instead.